### ISSUE TITLE
Build 9.16.1 broke firebase deploy command

### ISSUE THREAD

**Issue by ljle** [firebase/firebase-tools#3664]
### [REQUIRED] Environment info

**firebase-tools:** 9.16.1

**Platform:** Windows through WSL with Arch Linux

### [REQUIRED] Test case

There's no minimal test case as the bug is related to the deploy function.

### [REQUIRED] Steps to reproduce

- Deploy any cloud function.

### [REQUIRED] Expected behavior

- The function(s) are deployed and a successfull deployment message appears.

### [REQUIRED] Actual behavior

- Deployment "seems" to go through but at the end of the process an error is printed out. This happens both on CI or by issuing the command manually from the CLI.
![image](https://user-images.githubusercontent.com/696474/129108881-e6db5ed1-e796-47a9-b8d9-ccf8c2773a18.png)

**Comment by JarrydMartin** [comment]
I am having the same issue.

When running firebase deploy with the debug flag I get the following error...

```
[2021-08-11T22:00:03.195Z] TypeError: track_1.default is not a function
    at release (/Users/jarryd/.nvm/versions/node/v14.17.3/lib/node_modules/firebase-tools/lib/deploy/functions/release.js:98:20)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
```

**Comment by bkendall** [comment]
Looks like it. Taking a look...

**Comment by nternouski** [comment]
Same here

**Comment by taeold** [comment]
We are making a hotfix to address this.

In the meantime, please revert your Firebase CLI to the previously working version:

```
$ npm i -g firebase-tools@9.16.0
```

**Comment by bkendall** [comment]
Should be fixed in `9.16.2` now!
